### ISSUE TITLE
Suggesion: Add option to select more than two reasons for declining a draft

### ISSUE THREAD

**Issue by asrgaohifd** [WPAFC/afch-rewrite#158]
There have been multiple instances where I have reviewed drafts that fall under more than two reasons for declining a draft and it would be helpful to select more than two.

**Comment by primefac** [comment]
There should always be one (or two) main reasons for declining. For example, a draft that fails `prof` likely also fails `v`, `rs`, and/or `ilc`, and those reasons are already included in prof's decline reason, so there's not much purpose in also declining as those. In other words, I do not see a reasons to include more than one decline rationale (if the draft needs that much work, it's probably better to add a note anyway).

**Comment by enterprisey** [comment]
Yeah, I originally figured you can have two reasons of equal importance, but when you get to three reasons one of them will be less important than the other two. Either way, I would be happier if there were consensus, or at least someone else agreeing with you, on [WT:WPAFC](https://en.wikipedia.org/wiki/Wikipedia_talk:WikiProject_Articles_for_creation).
